tests/unit/test_discourse.py

```diff
@@ -112,6 +115,12 @@
             None,
             id="valid trailing /",
         ),
+        pytest.param(
+            "/t/slug/1",
```

Comment:
Should use topic url prefix constant

Reply:
Thanks, I have updated it here an in a bunch of other places as well, should be available shortly

Reply:
Fixed, same for the other comments